onadata/apps/api/management/commands/delete_users.py

```diff
@@ -0,0 +1,117 @@
+"""
+Delete users management command.
+"""
+import sys
+from django.contrib.auth.models import User
+from onadata.apps.logger.models import XForm
+from onadata.apps.logger.models import Instance
+from onadata.apps.logger.models import Project
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import timezone
+
+
+def get_user_object_stats(
+        username):  # pylint: disable=R0201
```

Comment:
A bit confused about why the `[R0201](http://pylint-messages.wikidot.com/messages:r0201)` is raised here.... ?